lua/oil/config.lua

```diff
@@ -28,6 +28,8 @@ local default_config = {
   },
   -- Send deleted files to the trash instead of permanently deleting them (:help oil-trash)
   delete_to_trash = false,
+  -- Set the default mode to create files (:help oil-file_creation)
+  create_files_mode = 420,
```

Comment:
We don't expect most users to need to modify this, so let's move it down to below the `view_options`.

Reply:
If we're adding the ability to adjust the mode of the created files, we should do the same for directories. We could use two options for this `new_file_mode` and `new_dir_mode`.

Reply:
Since most people are used to thinking about file permissions in octal, the config option should be octal and we should automatically convert it.

Reply:
ok, sounds good